cmake/Modules/FindACL.cmake

```diff
@@ -0,0 +1,61 @@
+# ----------
+# FindACL
+# ----------
+#
+# Finds the Arm Compute Library
```

Comment:
Does this file exists somewhere else? If so, please reference where it was copied from
If this is create exclusively for PyTorch, please use modern CMAKE, i.e. instead of (or in addition to) defining global variables add libraries/targets, something like
```cmake
    add_library(ArmComputeLib INTERFACE)
    target_link_libraries(ArmComputeLib INTERFACE ${ACL_LIBRARIES})
    target_include_directories(ArmComputeLib INTERFACE ${ACL_INCLUDE_DIRS})
```

Reply:
This file was not exclusively create for PyTorch, it was copied from oneDNN: https://github.com/oneapi-src/oneDNN/blob/main/cmake/FindACL.cmake

I referenced that in the new ghstack PR - see [this line](https://github.com/pytorch/pytorch/pull/148584/files#diff-4b23b04960eafa07ab84df9d9ba346a133d523e285ed0df3655efcfcd56bdcf1R1)